test/src/components/Modal-test.js

```diff
@@ -30,12 +25,12 @@ describe('Modal', () => {
   it('should show modal', () => {
     var renderedModal = render(<Modal />).first();
     renderedModal.showModal();
-    renderedModal.elm.modal.called.should.equal(true);
+    renderedModal.state.isShown.should.equal(true);
   });
```

Comment:
This test is for `showModal` testing that `it('sets isShown true)`.  The description should be updated as it says that it is testing that the the modal is shown.